ember-composable-helpers/src/helpers/map-by.ts

```diff
@@ -1,9 +1,9 @@
 import { helper } from '@ember/component/helper';
 import { get } from '@ember/object';
 import { isEmpty } from '@ember/utils';
-import asArray from '../utils/as-array';
+import asArray from '../utils/as-array.ts';
 
-export function mapBy([byPath, array]) {
+export function mapBy<T extends object>([byPath, array]: [keyof T, T[]]) {
```

Comment:
`get` types are slightly different with built-in types--would it be preferable to use a type assertion on the `get` args instead of modifying the generic constraint like I've done here?

Reply:
probably, ya